tests/trezor/trezor_tests.cpp

```diff
@@ -426,6 +509,52 @@ static crypto::public_key get_tx_pub_key_from_received_outs(const tools::wallet2
   }
 }
 
+static void generate_tx_block(std::vector<test_event_entry>& events, std::list<cryptonote::transaction> &txs_lst, cryptonote::block & head,
+                              const cryptonote::account_base& from, const cryptonote::account_base& to, uint64_t amount, size_t repeat,
+                              size_t mixin, bool rct, rct::RangeProofType range_proof_type, int bp_version)
+{
+  const size_t per_tx_limit = 14;
+  const size_t iters = std::ceil(repeat / (double)per_tx_limit);
```

Comment:
If `repeat` is 0, `iters` will also be 0. I would do `repeat ? std::ceil(repeat / (double)per_tx_limit) : 1` to guarantee at least 1 iteration of the loop.